dev/sbt-plugin/src/main/scala/com/lightbend/lagom/sbt/LagomPlugin.scala

```diff
@@ -405,13 +413,37 @@ object LagomPlugin extends AutoPlugin {
   }
 
   private def runAllMicroservicesTask: Initialize[Task[Unit]] = Def.taskDyn {
+
     val projects = microservicesProjects.value
-    val filter = ScopeFilter(inProjects(projects: _*))
+    val filter: ScopeFilter = ScopeFilter(inProjects(projects: _*))
+    // check if application.conf is present in service
+    val serviceResourceDirectories = ((unmanagedResourceDirectories in Runtime) ?? Nil).all(filter)
+    //Verifying the presence of the required application.conf file
+    val verifyApplicationConfTask = Def.task {
+
+      val servicesPaths: Seq[File] = serviceResourceDirectories.value.flatten
```

Comment:
You are rigth. I used flatten with in mind that only the standard resources directory will be into ressourcesdirectories' setting. I updated my code by removing flatten. Now, I take a benefit of the implicit method that convert a sequence of files to a pathfinder.